python/pyarrow/array.pxi

```diff
@@ -2363,6 +2363,78 @@ cdef class MapArray(ListArray):
         Returns
         -------
         map_array : MapArray
+
+        Examples
+        --------
+        First, let's understand the structure of our dataset when viewed in a rectangular data model. 
+        The total of 5 respondents answered the question "How much did you like the movie x?".
+        The value -1 in the integer array means that the value is missing. The boolean array
+        represents the null bitmask corresponding to the missing values in the integer array.
+
+        >>> movies_rectangular = np.ma.masked_array([
+        >>>     [10, -1, -1],
+        >>>     [8, 4, 5],
+        >>>     [-1, 10, 3],
+        >>>     [-1, -1, -1],
+        >>>     [-1, -1, -1]
+        >>> ],
+        >>> [
+        >>>     [False, True, True],
+        >>>     [False, False, False],
+        >>>     [True, False, False],
+        >>>     [True, True, True],
+        >>>     [True, True, True],
+        >>> ])
```

Comment:
```suggestion
        ...     [10, -1, -1],
        ...     [8, 4, 5],
        ...     [-1, 10, 3],
        ...     [-1, -1, -1],
        ...     [-1, -1, -1]
        ... ],
        ... [
        ...     [False, True, True],
        ...     [False, False, False],
        ...     [True, False, False],
        ...     [True, True, True],
        ...     [True, True, True],
        ... ])
```

I think this should fix the failing build due to docstest error. SImilar should be done bellow (when the code breaks into multiple lines, `...` should be used instead of `>>>`.